### PR TITLE
[receiver/kafkareceiver] support zipkin_json(v1)

### PR DESCRIPTION
**Pull request by Subhasish-Behera**
**Description:** <Describe what has changed.>
<pre>
changes made are :
zipkin_unmarshaler.go
    1. added new function newZipkinJSONv1Unmarshaler
    2. added const zipkinJSONv1Encoding 
zipkin_marshaler_test.go
    1. commented possible test
unmarshaler.go
    1.added zipkinJSONv1 to defaultTracesUnmarshalers() 
unmarshaler_test.go
    1.updated tests
</pre>

**Link to tracking Issue:** <Issue number if applicable>
#14472
**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by MovieStoreGuy — COMMENTED** [review]
There is a few extra things you'll need to do:

- [x] Add in a change log as an enhancement for the kafka receiver  (run `make chlog-new`)
- [ ] Update the README to say this is now an exported type to use.
- [ ] Ensure the changes are part of the test.

File: receiver/kafkareceiver/zipkin_unmarshaler_test.go [hunk 1]
```diff
@@ -89,6 +95,12 @@ func TestUnmarshalZipkin(t *testing.T) {
 			bytes:       thriftTransport.Buffer.Bytes(),
 			expected:    tdThrift,
 		},
+		//{
+		//	unmarshaler: newZipkinJSONv1Unmarshaler(),
+		//	encoding:    "zipkin_v1json",
+		//	bytes:       jsonv1Bytes,
+		//	expected:    td,
+		//},
```
Comment (MovieStoreGuy) [inline receiver/kafkareceiver/zipkin_unmarshaler_test.go:103]: I would prefer not having commented out code in the test, is there a reason for having this?

File: receiver/kafkareceiver/zipkin_unmarshaler_test.go [hunk 2]
```diff
@@ -112,6 +124,12 @@ func TestUnmarshalZipkinJSON_error(t *testing.T) {
 	assert.Error(t, err)
 }
 
+//func TestUnmarshalZipkinJSONv1_error(t *testing.T) {
+//	p := newZipkinJSONv1Unmarshaler()
+//	_, err := p.Unmarshal([]byte("+$%"))
+//	assert.Error(t, err)
+//}
```
Comment (MovieStoreGuy) [inline receiver/kafkareceiver/zipkin_unmarshaler_test.go:131]: Uncomment this and add it to the test being run.

**Comment by MovieStoreGuy** [comment]
Hey @Subhasish-Behera ,

are you still working on this?

**Comment by Subhasish-Behera** [comment]
https://cloud-native.slack.com/archives/CJFCJHG4Q/p1671807742419309?thread_ts=1671515726.873829&cid=CJFCJHG4Q
Yep,I would like to complete it but that doesn't seems possible as `fromtranslator` is missing from `zipkinv1` unlike `zipkinv2`.

**Comment by Subhasish-Behera** [comment]
@MovieStoreGuy ,@pavolloffay,@TylerHelmuth not sure why this PR was closed. I want to start working on the `fromtranslator` file(not currently present) for `zipkinv1`. Should I make a new branch for that or continue working on this branch? 
(The @-mentions are for the assigned reviewers)

**Pull request closed without merging**